docs_guidelines/tutorials/new_doc_project.rst

```diff
@@ -0,0 +1,610 @@
+      tools/ignore_untagged_notes.sh
+      sphinx-build -b html docs/ docs/_build/html
+      tools/deploy_documentation.sh
+    shell: bash
+
+So the final version of this step is:
+
+.. code-block:: yaml
+
+    - name: Build and publish
+      env:
+        encrypted_rclone_key: ${{ secrets.encrypted_rclone_key }}
+        encrypted_rclone_iv: ${{ secrets.encrypted_rclone_iv }}
+        QISKIT_PARALLEL: False
+        QISKIT_DOCS_BUILD_TUTORIALS: 'always'
+      run: |
+        echo "earliest_version: 0.1.0" >> releasenotes/config.yaml
+        tools/ignore_untagged_notes.sh
+        sphinx-build -b html docs/ docs/_build/html
+        tools/deploy_documentation.sh
+      shell: bash
+
+
+The complete ``.github/workflows/deploy-docs.yml`` is then:
+
+.. code-block:: yaml
+
+    # This code is part of Qiskit.
+    #
+    # (C) Copyright IBM 2022.
+    #
+    # This code is licensed under the Apache License, Version 2.0. You may
+    # obtain a copy of this license in the LICENSE.txt file in the root directory
+    # of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+    #
+    # Any modifications or derivative works of this code must retain this
+    # copyright notice, and modified files need to carry a notice indicating
+    # that they have been altered from the originals.
+
+    name: Deploy Docs
+
+    on:
+      workflow_dispatch:
+
+    concurrency:
+      group: ${{ github.repository }}-${{ github.ref }}-${{ github.head_ref }}-${{ github.workflow }}
+      cancel-in-progress: true
+
+    jobs:
+      docs_publish:
+        if: ${{ startsWith(github.ref, 'refs/heads/stable') && contains('["mantainer-1","mantainer-2", ...]', github.actor) }}
+        runs-on: ubuntu-latest
+        strategy:
+        matrix:
+            python-version: [3.8]
+        steps:
+        - uses: actions/checkout@v3
+            with:
+            fetch-depth: 0
+        - uses: actions/setup-python@v4
+            with:
+            python-version: ${{ matrix.python-version }}
+        - name: Install X
+            run: |
+              pip install -e .
+              pip install -U -c constraints.txt -r requirements-dev.txt
+            shell: bash
+        - name: Install Dependencies
+            run: |
+              pip install packages
+              sudo apt-get install -y other_packages
+            shell: bash
+        - name: Build and publish
+            env:
+              encrypted_rclone_key: ${{ secrets.encrypted_rclone_key }}
+              encrypted_rclone_iv: ${{ secrets.encrypted_rclone_iv }}
+              QISKIT_PARALLEL: False
+              QISKIT_DOCS_BUILD_TUTORIALS: 'always'
+            run: |
+              echo "earliest_version: 0.1.0" >> releasenotes/config.yaml
+              tools/ignore_untagged_notes.sh
+              sphinx-build -b html docs/ docs/_build/html
+              tools/deploy_documentation.sh
+            shell: bash
+
+
+You have created and deployed a new Qiskit documentation project!
```

Comment:
After adding these files, how will the documentation get built and publish on qiskit.org? Just need to manually trigger the `Docs Publish` workflow? And where is the url configured? For example, how does the documentation of Qiskit Nature gets to publish under `qiskit.org/documentation/nature`?

Reply:
From my understanding you only need to manually trigger the workflow. The idea is that with the workflow you automatically update the contents of `IBMCOS:qiskit-org-web-resources/documentation/x`, from where `qiskit.org/documentation/x` is built. However, configuring that (`IBMCOS:qiskit-org-web-resources/documentation/x` $\rightarrow$ `qiskit.org/documentation/x`) is something that the Qiskit organization admins are supposed to do, not the owner of the repo (again, from my understanding). It's similar to the inclusion of the RClone keys.

Reply:
Now I mention that you also have to ask the Qiskit admins to set up the IBMCOS bucket and URL.